libdnf-cli/progressbar/multi_progress_bar.cpp

```diff
@@ -60,15 +60,38 @@ void MultiProgressBar::add_bar(std::unique_ptr<ProgressBar> && bar) {
     }
 
     bars_all.push_back(std::move(bar));
-    total.set_total(static_cast<int>(bars_all.size()));
+    if (total.get_total() < static_cast<int32_t>(bars_all.size())) {
+        total.set_total(static_cast<int32_t>(bars_all.size()));
```

Comment:
Minor: repeated expression `static_cast<int32_t>(bars_all.size())`.

Reply:
It is minor. But I will do change.